include/modem/sms.h

```diff
@@ -207,6 +217,34 @@ void sms_unregister_listener(int handle);
  */
 int sms_send_text(const char *number, const char *text);
 
+/**
+ * @brief Send SMS message.
```

Comment:
I don't know if we should break the API. I've thought that having a clear function for sending text would be nice although I realize that the use cases for such an operation in IOT device is limited.

In any case, maybe `sms_send_data()` could still be `sms_send()` because it will be the function to send all kinds of data (if we ever implement support for normal 8bit data).

Reply:
Renaming `sms_send_data()` to `sms_send()` sounds good to me. You can keep the `sms_send_text()` if you like, it's not a deal breaker for me. 😄

Reply:
Done. I'll keep `sms_send_text()` although I'm not sure I'm just too used to it and it wouldn't really be needed.

Reply:
Clarifications done to function descriptions of `sms_send_text()` and `sms_send()`.